go/src/koding/kites/config/config.go

```diff
@@ -104,6 +104,36 @@ func (u *URL) WithPath(paths ...string) *URL {
 	return &URL{URL: &ur}
 }
 
+// StringOrBool implements json.Unmarshaler, which can parse
+// either boolean type or converts string to bool.
+type StringOrBool bool
+
+var _ json.Unmarshaler = (*StringOrBool)(nil)
+
+func (b *StringOrBool) UnmarshalJSON(p []byte) error {
```

Comment:
under normal circumstances it should :) tbh we shouldnt need write our own marshaller for parsing bool, we could just use the string tag in json tags.  

that sounds like an input error to me. Where are you getting the config from? We may need to fix the places where we are writing to it instead of trying to adjust ourselves to it - if it is already not too late :)

Using "Unquote" could create problems where the input is `[]byte("true")` we can leave it as is